courses/free-courses-fa_IR.md

```diff
@@ -5,6 +5,7 @@
 * [Git](#Git)
 * [Java](#java)
 * [JavaScript](#javascript)
+* [Kotlin](#kotlin)
```

Comment:
:information_source: It's usually not helpful to add a category for a single resource. :twisted_rightwards_arrows: Try to...
- ... find another category matching their topics.
- ... see if there are other books in our lists or third-party that would fit into your category (this fact contributes to consolitate it).